src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__section">
+      <article class="stars stars--0">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--1">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--2">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--3">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--4">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+
+      <article class="stars stars--5">
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+        <div class="rating-star"></div>
+      </article>
+    </div>
```

Comment:
Remember about correct indentation between parent and child elements. There should be a 2-space offset for each level of nesting.